src/test/java/org/symphonyoss/symphony/messageml/elements/form/FormTest.java

```diff
@@ -156,5 +152,46 @@ public void testWithMultipleDialogs() throws Exception {
     assertTrue(presentationML.matches(expectedPattern));
   }
 
+  @Test
+  public void testMultiSubmitReset() throws InvalidInputException, IOException, ProcessingException {
```

Comment:
if we want to be super complete we would test it with no-reset too given there are few valid values (but I'm ok as is)